twingate/internal/client/pagination_test.go

```diff
@@ -0,0 +1,84 @@
+package client
```

Comment:
Why is this here and not in `twingate/internal/test/client` ?

Reply:
here I'm testing private func `fetchPages` and it's not accessible from other packages